db/db.go

```diff
@@ -87,6 +87,7 @@ const (
 	CLevelDBBackendStr  = "cleveldb"
 	GoLevelDBBackendStr = "goleveldb"
 	MemDBBackendStr     = "memdb"
+	FSDBBackendStr      = "fsdb" // using the filesystem naively
```

Comment:
why do need it? just curios

Reply:
We had similar code in go-crypto/keys/* to save files in the filesystem (or a mock memory system for testing).  With this we can use the DB interface, instead of everyone writing their own unit tests to test io stuff for their custom X-manager  e.g. key-manager).

Reply:
Ok, cool. Thanks for explaining this to me.